hudi-common/src/main/java/org/apache/hudi/common/util/ConfigUtils.java

```diff
@@ -78,7 +79,9 @@ public static String getOrderingField(Properties properties) {
    */
   public static String getPayloadClass(Properties properties) {
     String payloadClass = null;
-    if (properties.containsKey(HoodieTableConfig.PAYLOAD_CLASS_NAME.key())) {
+    if (properties.containsKey(HoodieTableConfig.PAYLOAD_TYPE.key())) {
+      payloadClass = RecordPayloadType.valueOf(properties.getProperty(HoodieTableConfig.PAYLOAD_TYPE.key())).getClassName();
```

Comment:
Not sure why we need this method separately, can we move it altogether into `RecordPayloadType`.